packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -71,20 +90,27 @@ export class Span implements api.Span, ReadableSpan {
     parentTracer: Tracer,
     context: Context,
     spanName: string,
-    spanContext: api.SpanContext,
-    kind: api.SpanKind,
+    spanContext: SpanContext,
+    kind: SpanKind,
     parentSpanId?: string,
-    links: api.Link[] = [],
-    startTime?: api.TimeInput,
-    clock: Clock = otperformance,
+    links: Link[] = [],
+    startTime?: TimeInput,
+    _deprecatedClock?: unknown, // keeping this argument even though it is unused to ensure backwards compatibility
   ) {
-    this._clock = clock;
     this.name = spanName;
     this._spanContext = spanContext;
     this.parentSpanId = parentSpanId;
     this.kind = kind;
     this.links = links;
-    this.startTime = timeInputToHrTime(startTime ?? clock.now());
+
+    const now = Date.now();
+    this._performanceStartTime = otperformance.now();
+    this._performanceOffset = now - this._performanceStartTime;
```

Comment:
Missing removal of timeOrigin ([basically opposite of this from last time](https://github.com/open-telemetry/opentelemetry-js/pull/3384#discussion_r1021740473)), meaning `this._performanceOffset` gets set to timeOrigin + drift

Then in _getTime: value + this._performanceOffset + inside hrTime additional timeOrigin -> sum in 2075

![image](https://user-images.githubusercontent.com/1122555/203386529-c83bd81e-d899-4c52-9b43-497d056f270e.png)

Reply:
bah thanks for catching